tilequeue/queue/grouper.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+from tilequeue.tile import coord_marshall_int
+
+
+CoordGroup = namedtuple('CoordGroup', 'coords metadata')
+
+
+class StartZoomGrouper(object):
+
+    """
+    group coords by a parent zoom tile, like z10
+
+    Take a sequence of coordinates, and emit a sequence of CoordGroup
+    instances. These contains coords grouped by their parent zoom, and
+    metadata that contains the parent zoom used.
+    """
+
+    def __init__(self, start_zoom_to_group):
+        self.start_zoom_to_group = start_zoom_to_group
+
+    def __call__(self, coords):
+        grouped = {}
```

Comment:
I found this a little hard to understand. Would it be more readable as:

```python
grouped = defaultdict(list)
for coord in coords:
    if coord.zoom >= self.start_zoom_to_group:
        # assuming zoomTo().container() is a no-op if the zooms are the same?
        coord_at_group_zoom = coord.zoomTo(self.start_zoom_to_group).container()
        group_key = coord_marshall_int(coord_at_group_zoom)
        grouped[group_key].append(coord)
    else:
         ...
```

?

Reply:
Sure, happy to move to using a `defaultdict` here instead.

Reply:
Updated in 4d2f306.